### PR TITLE
Translation update for da_DK locale

### PR DESCRIPTION
**Pull request by tgc**
Translation update from the Royal Danish Library.

Not all keys have been translated. Focus has been on the user visible parts and less on the technical parts that only admins see.

### CODE REVIEW

**Comment by tgc** [comment]
The pull request is now updated with a full translation from @neffe

File: locale/da_DK/common.xml [hunk 1]
```diff
@@ -542,4 +522,27 @@
...
+	<message key="notification.addedReviewerNoEmail">{$reviewerName} blev tilføjet som bedømmer på denne indsendelse og har ikke fået besked via mail.</message>
+	<message key="notification.reviewerThankedEmail">Tak, e-mail er nu sendt til bedømmer.</message>
+	<message key="notification.sendNotificationConfirmation">Send meddelelse via mail til alle registrerede brugere</message>
+	<message key="notification.editedNavigationMenuItem">Navigationsmenu-elementerne blev opdateret</message>
+	<message key="notification.removedNavigationMenuItem">Navigationsmenu-elementerne blev fjernet</message>
+	<message key="notification.addedNavigationMenuItem">Navigationsmenu-elementerne blev tilføjet</message>
+	<message key="notification.addedNavigationMenu">Navigation menu was succesfully added</message>
+	<message key="notification.editedNavigationMenu">Navigationsmenu blev opdateret</message>
+	<message key="notification.removedNavigationMenu">Navigationsmenu blev fjernet</message>
+	<message key=""><![CDATA[]]></message>
```
Comment (asmecher) [inline locale/da_DK/common.xml]: These empty keys should be removed.

Comment (asmecher) [reply locale/da_DK/common.xml]: (Done)

**Comment by asmecher** [comment]
Thanks, @tgc and @neffe! I've merged this, cherry-picked forward to the `master` branch, and made a few tweaks (updating copyright dates, changing line endings from DOS to UNIX).

**Pull request merged**